Content.IntegrationTests/Tests/Preferences/ServerDbSqliteTests.cs

```diff
@@ -62,6 +62,7 @@ private static HumanoidCharacterProfile CharlieCharlieson()
                 },
                 PreferenceUnavailableMode.StayInLobby,
                 new List<string> (),
+                new List<string> (),
                 new List<string>()
             );
```

Comment:
Please use named args, and even better just make it nullable if no data is valid to save the allocation.